ds3-sdk/src/test/java/com/spectralogic/ds3client/Ds3ClientBuilder_Test.java

```diff
@@ -31,6 +32,27 @@ public void createBasicClient() throws Exception {
     }
 
     @Test
+    public void createBasicClientFromENV() throws Exception {
+        try {
+            final Ds3ClientBuilder builder = Ds3ClientBuilder.fromEnv();
```

Comment:
If the variables aren't setup, it should check the system and confirm they aren't setup and pass.  If they are setup, it should find them and pass.  I'm checking the wrong variables though . . .